Mathlib/MeasureTheory/Constructions/BorelSpace/Real.lean

```diff
@@ -465,6 +466,81 @@ _root_.measurable_of_tendsto_nnreal := NNReal.measurable_of_tendsto
 
 end NNReal
 
+namespace EReal
+
+lemma measurableEmbedding_coe : MeasurableEmbedding Real.toEReal :=
+  isOpenEmbedding_coe.measurableEmbedding
+
+lemma lowerSemicontinuous_add : LowerSemicontinuous fun p : EReal × EReal ↦ p.1 + p.2 := by
```

Comment:
Could you move this to a better place? This has nothing to do with measure theory.